File: Smallest multiple.py
i=1
j = 0
n = 0
count =0
n = int(input("Enter divided by upto which number "))
while(i!=0):
    count = 0;
    for j in range (1,n+1):
        if(i%j==0):
            count = count + 1
    if(count == n):
        print(i)
        break
    i=i+1

File: Smallest multiple.cpp
#include <iostream>
using namespace std;
int main()
{
    long int i=1,j,n,count;
    cout << "Enter divided by upto which number ";
    cin >>n;
    while(i!=0)
    {
        count = 0;
        for(j=1;j<=n;j++)
        {
            if(i%j==0)
            {
                count++;
            }
        }
        if(count == n)
        {
            cout << i;
            break;
        }
        i++;
    }
}
